lib/ddtrace/profiling.rb

```diff
@@ -14,20 +14,14 @@ def unsupported_reason
       # NOTE: Only the first matching reason is returned, so try to keep a nice order on reasons -- e.g. tell users
       # first that they can't use this on JRuby before telling them that they are missing protobuf
 
-      ruby_engine_unsupported? || ruby_version_unsupported? ||
-        protobuf_gem_unavailable? || protobuf_version_unsupported? || protobuf_failed_to_load?
+      ruby_engine_unsupported? || protobuf_gem_unavailable? || protobuf_version_unsupported? || protobuf_failed_to_load?
     end
 
     def self.ruby_engine_unsupported?
       'JRuby is not supported' if RUBY_ENGINE == 'jruby'
     end
     private_class_method :ruby_engine_unsupported?
```

Comment:
Minor: I'll do a few post-2.0 cleanups as well, but we can finally use do `private_class_method def ...` and avoid having to retype the method name afterwards! :) :) :)

Reply:
👍